.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
You might want to update the gitignore to include the .vscode directory. This shouldn't be a shared folder/file set.